core/test/unit/data/exporter/index_spec.js

```diff
@@ -94,8 +94,8 @@ describe('Exporter', function () {
 
             // Execute
             exporter.doExport({include: ['clients', 'client_trusted_domains']}).then(function (exportData) {
-                // all tables, except of the tokens
-                const expectedCallCount = schemaTables.length - 2;
+                // all tables, except of the tokes and sessions
+                const expectedCallCount = schemaTables.length - (exporter.EXCLUDED_TABLES.length - 2);
```

Comment:
Same as above, and here the 2 comes from - `['clients', 'client_trusted_domains']`